users/views.py

```diff
@@ -1,3 +1,60 @@
-from django.shortcuts import render
+import requests
+import jwt
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+from users.models import User
+
+from freepass import settings
+
+class KakaoAPI:
+    def __init__(self):
+        self.kakao_token_url = "https://kauth.kakao.com/oauth/token"
+        self.kakao_user_url  = "https://kapi.kakao.com/v2/user/me"
+
+    def get_kakao_token(self, auth_code):
+        data = {
+            "grant_type"  : "authorization_code",
+            "client_id"   : settings.KAKAO_APP_KEY,
+            "redirect_uri": settings.REDIRECT_URI,
+            "code"        : auth_code
+        }
+        access_token = requests.post(self.kakao_token_url, data=data).json()['access_token']
+        return access_token
```

Comment:
1. requests에 timeout 속성을 추가해 주세요. http는 요청을 보낸 후 응답이 올 때까지 작업을 멈춥니다. 따라서 작업이 멈추는 퇴대 시간인 timeout을 안 줄 경우 시스템이 멈출 수 있습니다.
2. requests에서 바로 데이터를 뽑는 것이 아니고 reponse에 담아 해당 response에 대한 평가가 필요합니다.
3. reponse를 분석하여 적절한 예외 처리를 추가해 주세요.
```suggestion
        response = requests.post(self.kakao_token_url, data=data, timeout=3)
        access_token = response.json()['access_token']
        return access_token
```